Build failure on FreeBSD related to lightning local backend

## Bug Report

This seems to have been introduced by 9cf69a9633a / #42733

```
% gmake server
CGO_ENABLED=1 GO111MODULE=on go build  -tags codes  -ldflags '-X "github.com/pingcap/tidb/parser/mysql.TiDBReleaseVersion=v6.7.0-alpha-548-g2002ca106e" -X "github.com/pingcap/tidb/util/versioninfo.TiDBBuildTS=2023-04-13 06:23:19" -X "github.com/pingcap/tidb/util/versioninfo.TiDBGitHash=2002ca106e99f56f6f06b04825c0b691267392df" -X "github.com/pingcap/tidb/util/versioninfo.TiDBGitBranch=master" -X "github.com/pingcap/tidb/util/versioninfo.TiDBEdition=Community" ' -o bin/tidb-server ./tidb-server
# github.com/pingcap/tidb/br/pkg/lightning/backend/local
br/pkg/lightning/backend/local/local_unix.go:33:25: undefined: RlimT
br/pkg/lightning/backend/local/local_unix.go:42:36: undefined: RlimT
br/pkg/lightning/backend/local/local_unix.go:49:12: undefined: RlimT
br/pkg/lightning/backend/local/local_unix.go:90:3: undefined: zapRlimT
gmake: *** [Makefile:150: server] Error 1
```

This is on FreeBSD 13.2-RELEASE

--- a/br/pkg/lightning/backend/local/local_freebsd.go
+++ b/br/pkg/lightning/backend/local/local_freebsd.go
@@ -20,8 +20,9 @@ import (
 	"go.uber.org/zap"
 )
 
-type Rlim_t = int64
+// RlimT is the type of rlimit values.
+type RlimT = int64
 
-func zapRlim_t(key string, val Rlim_t) zap.Field {
+func zapRlimT(key string, val RlimT) zap.Field {
 	return zap.Int64(key, val)
 }

